### PR TITLE
Added `from e3nn.o3.irrep import lXY ` syntax sugar

### PR DESCRIPTION
**Pull request by dmadisetti**
In your MRS talk, and going through the examples. I thought I would suggest this quick feature similar to `sympy`'s var: https://docs.sympy.org/latest/modules/core.html#var

## Description

Here, we can generate an atom by import opposed to string creation. Example:

old:
```python
from e3nn.o3 import  Irrep
l3o = Irrep("3o")
```

new:
```python
from e3nn.o3.irrep import l3o
```

## Motivation and Context
This syntax is a little less bulky, but is totally not needed. Feel free to reject this change.

## How Has This Been Tested?
Appended to test in irreps.creation. Let me know if you'd like more. Tested in colab, but hoping your CI will pick it up.

## Checklist:
- [x] I have read the [**CONTRIBUTING**](https://github.com/e3nn/e3nn/blob/main/CONTRIBUTING.md) document.
- ? My code follows the code style of this project.
- ? I have updated the documentation (if relevant).
- [x] I have added tests that cover my changes (if relevant).
- [x] The modified code is cuda compatible (github tests don't test cuda) (if relevant).
- [x] I have updated the [Changelog](https://github.com/e3nn/e3nn/blob/main/ChangeLog.md).

### CODE REVIEW

**Review by mariogeiger — APPROVED** [review]

**Comment by dmadisetti** [comment]
Thanks for the flake!

**Comment by dmadisetti** [comment]
I'll take a look at this. Sorry about it breaking, that's a bit embarrassing. I'll check 3.9, 3.7 is what I tested on

**Comment by dmadisetti** [comment]
Solve! I think. `pytest` does some weird module attribute spoofing stuff. Using `__getattribute__` over `__getattr__` allows for pytest to fallback (and is better anyway).

File: tests/o3/irreps_test.py [hunk 1]
```diff
@@ -19,6 +19,8 @@ def test_creation():
     o3.Irreps([(16, "3e"), '1e'])
     o3.Irreps([(16, "3e"), '1e', (256, (1, -1))])
 
+    assert o3.irreps.l3e == ir
```
Comment (mariogeiger) [inline tests/o3/irreps_test.py:22]: Is it `o3.irreps` or `o3.irrep` ? `o3.irrep` looks better to me

**Comment by dmadisetti** [comment]
Great, I'm also going to make a PR suggesting one runs `flake8 . --count --show-source --statistics` since this has tripped me up a few times.

**Comment by dmadisetti** [comment]
Ok, I figure out testing, and I figured out flake.

This is still broken, so I'm turning this to draft. I will fix it, but it's not as simple as I thought with pytest

**Comment by dmadisetti** [comment]
Ok tests work locally. Sorry for taking so long on such a silly and small change

File: tests/o3/irreps_test.py (hunk 1 above)
Comment (dmadisetti) [reply tests/o3/irreps_test.py]: irrep it is!

**Comment by mariogeiger** [comment]
Let me give up the support of python 3.6

**Pull request merged**